Readme.md

```diff
@@ -5,11 +5,15 @@
     </picture>
 </div>
 
-[![](https://img.shields.io/badge/docs-dev-blue.svg)](https://juliamanifolds.github.io/Manifolds.jl/latest/)
+[![](https://img.shields.io/badge/docs-dev-blue.svg)](https://juliamanifolds.github.io/LieGroups.jl/dev/)
 [![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle) [![CI](https://github.com/JuliaManifolds/LieGroups.jl/actions/workflows/ci.yml/badge.svg)](https://github.com/JuliaManifolds/LieGroups.jl/actions?query=workflow%3ACI+branch%3Amain) [![codecov.io](http://codecov.io/github/JuliaManifolds/LieGroups.jl/coverage.svg?branch=main)](https://codecov.io/gh/JuliaManifolds/LieGroups.jl/)
 
 This is a package to rework the Lie group features of [`Manifolds.jl`](https://juliamanifolds.github.io/Manifolds.jl/stable/) in a unified way into a separate package.
 
+> [!NOTE]
+> Since this is a rework of the features from [`Manifolds.jl`](https://juliamanifolds.github.io/Manifolds.jl/stable/), both `LieGroups.jl` and `Manifolds.jl` 0.10 export a few types. For that reason it's not recommended to load both packages in a productive environment, unless you are aware, which variables you want to import wherefrom.
```

Comment:
```suggestion
> Since this is a rework of the features from [`Manifolds.jl`](https://juliamanifolds.github.io/Manifolds.jl/stable/), both `LieGroups.jl` and `Manifolds.jl` 0.10 export a few types. For that reason it's not recommended to load both packages in a productive environment, unless you are aware, which variables you want to import wherefrom.
```
I'd just say any production use of LieGroups.jl is not recommended until Manifolds.jl 0.11 is out.

Reply:
I would not mind anyone using this earlier, just not together with Manifolds.jl (before 0.11).

Reply:
Manifolds.jl is literally a requirement of LieGroups.jl so it's not really possible :slightly_smiling_face:

Reply:
It is. As long as you just do

`using LieGroups`


all is fine, since LieGroups only exports its own `Identity` type for example but if you currently do

`using Manifolds, LieGroups` both export `identity` and you are doomed. That is what is meant here.

Reply:
Then I'd suggest clarifying that you mean explicit import. Manifolds.jl is still loaded, the only difference is whether symbols are added to the module or not.

Reply:
Well it already states that it is due to “exporting a few types” – maybe more precise would be “a few types of common name which leads to a naming conflict”.  But I can think about how to rephrase that sure.

Reply:
Thanks, it is much better now :+1: